projects/snake/index.js

```diff
@@ -50,14 +33,51 @@ async function snakeGenesisTVL(api) {
 }
 
 
+const tvl = async (api) => {
+  const balances = {}
+
+  const [
+    snakeLpBal, // snake-s
+    gsnakeLpBal, // gsnake-s
+    shadowLpBal, // shadow-s
+    mclbLpBal, // mclb-s
+    shedewLpBal // shadow-shedew
+  ] = await api.multiCall({
+    calls: [
+      { target: "0x28c6d651A9592CE61fdba8fffEFFcD0d33023B8A", params: "0xFE6915a0983a304F4D131DA635664030dA06Bcd2", },
+      { target: "0xfc9E8be44d148242075dcCdbA42b81C7A8DF9E06", params: "0xFE6915a0983a304F4D131DA635664030dA06Bcd2", },
+      { target: "0xCBcAd939E2bbbe01850A141F204f25DF63b8FC5B", params: "0xFE6915a0983a304F4D131DA635664030dA06Bcd2", },
+      { target: "0x924f905B084252ACAD9E8AA5746531ae54698439", params: "0xFE6915a0983a304F4D131DA635664030dA06Bcd2", },
+      { target: "0x8279C4F93C385C7DA440b6140Ec2811664BeB832", params: "0xFE6915a0983a304F4D131DA635664030dA06Bcd2", },
+    ],
+    abi: "erc20:balanceOf",
+  });
+
+  sdk.util.sumSingleBalance(balances, "0x287c6882dE298665977787e268f3dba052A6e251", snakeLpBal)
+  sdk.util.sumSingleBalance(balances, "0xb901D7316447C84f4417b8a8268E2822095051E6", gsnakeLpBal)
+  sdk.util.sumSingleBalance(balances, "0xF19748a0E269c6965a84f8C98ca8C47A064D4dd0", shadowLpBal)
+  sdk.util.sumSingleBalance(balances, "0xDE87419F0774eef80b0EbD134Ff54ece0d4927C9", mclbLpBal)
+  sdk.util.sumSingleBalance(balances, "0x880ADDfEB13bb647a2dc5fC01f5ADCbB6Baa401F", shedewLpBal)
+
+  api.add('0x287c6882dE298665977787e268f3dba052A6e251', snakeLpBal)
+  api.add('0xb901D7316447C84f4417b8a8268E2822095051E6', gsnakeLpBal)
+  api.add('0xF19748a0E269c6965a84f8C98ca8C47A064D4dd0', shadowLpBal)
+  api.add('0xDE87419F0774eef80b0EbD134Ff54ece0d4927C9', mclbLpBal)
+  api.add('0x880ADDfEB13bb647a2dc5fC01f5ADCbB6Baa401F', shedewLpBal)
+
+  return sumUnknownTokens({  useDefaultCoreAssets: true, api, resolveLP: true,  });
+};
+
+
 module.exports = {
   methodology: "Pool2 deposits consist of SNAKE/S and GSNAKE/S LP tokens deposits while the staking TVL consists of the GSNAKEs tokens locked within the Masonry contract(0x5A5d34826ab31003F26F8A15e9B645803d85eA81).",
   sonic: {
-    // tvl: snakeGenesisTVL,
-    tvl: async () => ({}),
-    // pool2,
-    // staking,
-    pool2: sumTokensExport({ owner: gsnakeRewardPoolAddress, tokens: ftmLPs, }),
+    tvl: sdk.util.sumChainTvls([
+      snakeGenesisTVL,
+      tvl,
+      staking(masonryAddress, gsnakeTokenAddress),
```

Comment:
@chimpydev  thanks for the PR, but this doublecounts staking now, fixed it in my PR

Reply:
Very appreciated! thanks for the help